src/android/MyConnectionService.java

```diff
@@ -61,14 +61,7 @@ public void onAnswer() {
                 // TelecomManager tm = (TelecomManager) CordovaCall.getCordova().getActivity().getApplicationContext().getSystemService(Context.TELECOM_SERVICE);
                 // tm.showInCallScreen(false);
 
-                // Allow enough time for our app to open and register the answer callback
-                final Handler handler = new Handler();
-                handler.postDelayed(new Runnable() {
-                    @Override
-                    public void run() {
-                        CordovaCall.emitEvent("answer", new PluginResult(PluginResult.Status.OK, payloadString));
-                    }
-                }, 1000);
```

Comment:
This wait 1000ms before sending it is both unreliable and arbitrary (and introduces unnecessary delay). Am replacing this solution with something better.